src/app/components/result-box/result-box.component.ts

```diff
@@ -1,10 +1,10 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'app-result-box',
   templateUrl: './result-box.component.html',
   styleUrls: ['./result-box.component.scss']
 })
 export class ResultBoxComponent {
-
+  @Input() word: string = ''
```

Comment:
```suggestion
  @Input() word = ''
```
when assign a variable is not necessary to declare its type (its infer from the assignation)